### ISSUE TITLE
Sporadic function of the time triggered audio cue

### ISSUE THREAD

**Issue by RunningCl0ud** [jonasoreland/runnerup#339]
I have a Samsung GS5 and I downloaded Android wear because I have read that it could support my Pebble in some way. Could it be wear that is causing this? If I trigger Time audio cue in Audio cue settings it works? 
When it works, it really helps me in a race!

**Comment by RunningCl0ud** [comment]
Got rid of Android wear but that did not helped. The sound turned of after about 3 km. Tried to look over the settings  if there where any cross interferences that I could think of but no. Turned of the phone and took out the battery and rebooted. First run after that, all went well.

**Comment by RunningCl0ud** [comment]
I went out for a little run and then I stopped for a little photo session with my phone. I did not pause Runner Up. After that the audio cue was silent. Is this a bug?

**Comment by RunningCl0ud** [comment]
Today it went silent after almost exactly 4 km.

**Comment by nebmo** [comment]
Strange, I have never seen this behavior

**Comment by jonasoreland** [comment]
I've seen it a couple of times...total.
Maybe we could add more debugging printouts since you seem to be getting
this quite regularly....

**Comment by RunningCl0ud** [comment]
Yes it is a bit odd and I can not figure out what is triggering this behaviour. 
What do You mean Jonas? Is there anything I can do to help?
